torchci/lib/bot/mergeBot.ts

```diff
@@ -46,19 +47,28 @@ function mergeBot(app: Probot): void {
       await dispatchEvent("try-revert");
       await reactOnComment(ctx, "+1");
     }
+    else if (commentBody.match(rebaseCmdPat) && ctx.payload.comment.user.login == "clee2000") {
```

Comment:
```suggestion
    else if (commentBody.match(rebaseCmdPat) {
```

Reply:
I put this there to gatekeep so that I could test before rolling out to everyone